Profiles/ProfilesAPI.py

```diff
@@ -1,6 +1,66 @@
-from flask import Blueprint
+# Profile API here
+from flask import Blueprint, make_response, request
 import sys
-from db import db
+from db import db #importing db variable from db file
+from flask_cors import cross_origin
 sys.path.append("../")
 
-profiles_api = Blueprint("profiles", __name__)
\ No newline at end of file
+profiles_api = Blueprint("profiles", __name__)
+
+
+@profiles_api.route("/<int:id>", methods = ["GET"])
+@cross_origin(supports_credentials=True)
+def get_profile(id):
+    try:
+        data = {"name":db[id]['name'], "scores":db[id]['scores']}
```

Comment:
can just
`data = db[id]` 

cause if you POST name only (qn2), that person in the db doesnt have ['scores']. code will break.